website/content/gateway/api_reference/resources/authorizations/authorizations_methods/googlepay.md

```diff
@@ -24,14 +24,21 @@ The shared secret derived from the ephemeral public key and your private key
 ID assigned by Google. Prepend it with either `merchant:` or `gateway:` depending on ID type.
 {{% /description_details %}}
 
+{{% description_term %}}googlepay[3dsecure] {{% regex %}}dictionary{{% /regex %}}{{% /description_term %}}
+{{% description_details %}}
+See [Authentication: [3dsecure]](#authentication-3dsecure).
+{{% regex_optional %}}Optional{{% /regex_optional %}}
+
 {{% /description_list %}}
 {{% notice %}}
 **Notice**: Signing is required to use the `googlepay` payment method.
 
-**Notice**: An authorization made with `googlepay` is strongly authenticated (SCA in PSD2) if `authMethod` is `CRYPTOGRAM_3DS` and the Google Pay guidelines for SCA (link) have been followed. If authMethod is PAN_ONLY, a 3-D Secure flow is required for SCA.
+**Notice**: An authorization made with `googlepay` is strongly authenticated (SCA in PSD2) if `authMethod` is `CRYPTOGRAM_3DS` and the [Google Pay guidelines for SCA](https://developers.google.com/pay/api/android/guides/resources/sca) have been followed. If `authMethod` is `PAN_ONLY`, a 3-D Secure flow is required for SCA and the resulting ARes/RReq must be supplied in the `3dsecure` sub-dictionary.
```

Comment:
```suggestion
**Notice**: An authorization made with `googlepay` is strongly authenticated (SCA in PSD2) if `authMethod` is `CRYPTOGRAM_3DS` and the [Google Pay guidelines for SCA](https://developers.google.com/pay/api/android/guides/resources/sca) has been followed. If `authMethod` is `PAN_ONLY`, a 3-D Secure flow is required for SCA and the resulting ARes/RReq must be supplied in the `3dsecure` sub-dictionary.
```

Reply:
I retract this suggestion :slightly_smiling_face: 

![Screenshot from 2023-12-13 14-42-52](https://github.com/clearhaus/gateway-api-docs/assets/14069221/9541b665-5f8b-4b54-a92f-a60465a21cd3)

![Screenshot from 2023-12-13 14-43-13](https://github.com/clearhaus/gateway-api-docs/assets/14069221/13943832-9f1b-446e-b494-2d5cf88ec427)

![Screenshot from 2023-12-13 14-42-57](https://github.com/clearhaus/gateway-api-docs/assets/14069221/3c56d3b6-91c3-4839-b1fc-60eb6cfa3094)

![Screenshot from 2023-12-13 14-43-19](https://github.com/clearhaus/gateway-api-docs/assets/14069221/892133f0-e77c-4bc4-a3a6-774141a717c5)